### PR TITLE
Add contig-id argument to vcf subcommand

### LINKED ISSUE #1006: Add contig_id to tskit vcf CLI
There's currently no way to influence the contig ID used in VCF generation for the ``tskit vcf <ts>`` command.

Add a ``--contig-id/-c`` option which defaults to "1" in the [vcf](https://github.com/tskit-dev/tskit/blob/main/python/tskit/cli.py#L208) subcommand.

Add a few tests for this in [test_cli.py](https://github.com/tskit-dev/tskit/blob/main/python/tests/test_cli.py)

### PR DESCRIPTION
**Pull request by szhan**
Fixes #1006

### CODE REVIEW

**Comment by szhan** [comment]
The argument should only accept strings, right?

**Comment by jeromekelleher** [comment]
Looks good so far, but need to update the ``run_vcf`` function to use the new argument.

**Comment by jeromekelleher** [comment]
Looks good

- Do we have tests for the VCF output, and can we do something quick and simple to check the contig arg is working properly?
- Will need to squash and rebase

**Comment by szhan** [comment]
I'm trying to figure out how to refactor `test_vcf_default_values()` and `test_vcf_ploidy_short_args()`.

File: python/tests/test_cli.py [hunk 1]
```diff
@@ -263,29 +263,35 @@ def test_fasta_long_args(self):
         assert args.tree_sequence == tree_sequence
         assert args.wrap == 50
 
-    def test_vcf_default_values(self):
+    test_parameters_vcf_ploidy = (
```
Comment (benjeffery) [inline python/tests/test_cli.py:266]: What's the benefit of a variable here? Might as well just put it in the call to parametrize.

**Review by benjeffery — APPROVED** [review]
LGTM - one nit about the test params.

**Pull request merged**